sbioapputils/app_runner/yaml_automation.py

```diff
@@ -194,9 +194,10 @@ def _prune_jupyter(script_text):
 
 def _parse_input_python_v2(file: BytesIO, file_type: str = 'py', verbose: bool = False):
     script_text = file.getvalue().decode('ASCII')
+    stripped_script = ''
     if file_type=='py':
         stripped_script = _prune_script(script_text)
-    elif file_type=='ipynb':
+    elif file_type=='ipynb' or len(stripped_script)==0:
```

Comment:
won't len(stripped_script) always be 0 at this point?

Reply:
If file_type=='py' and _prune_script returns something, then its length will be greater. There was an issue where sometimes _prune_script returns an empty string (if neither argparse nor click are used in the script)

Reply:
but if the filetype is py, then the code will never reach this line

Reply:
Ah sorry, I've changed the elif to "if". It was correct in the other PR from last week, but I missed the mistake here